### PR TITLE
[Pebble Action] Correct linter errors only

### PR DESCRIPTION
**Pull request by bgilmer77**
Signed-off-by: Brad Gilmer <brad@gilmer.tv>

### CODE REVIEW

File: bundles/action/org.openhab.action.pebble/README.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 
 ## Configuration
 
 You can either configure the required Pebble token in `services/pebble.cfg` and refer to it by name, or provide it explicitly in your action calls.
 
-| Property | Default | Required | Description |
-|----------|---------|:--------:|-------------|
-| `<name>.token` | | | A token to use in action calls by name |
+| Property       | Default | Required | Description                            |
+|----------------|---------|-:-:------|----------------------------------------|
```
Comment (9037568) [inline bundles/action/org.openhab.action.pebble/README.md]: This is broken.

Comment (bgilmer77) [reply bundles/action/org.openhab.action.pebble/README.md]: @9037568 I believe my latest commit fixes this.  Please confirm, and if it is okay, please merge.  Thanks and Happy New Year.

Comment (9037568) [reply bundles/action/org.openhab.action.pebble/README.md]: Happy New Year to you!

However, I'm afraid the table is still broken.

**Comment by bgilmer77** [comment]
@9037568 finally figured out what the problem was with all of the tables.  The on-line tool I was using was putting the colons in the headers in the wrong place.  Please verify that this is now okay.

**Comment by bgilmer77** [comment]
@9037568, Chris I am still on the learning curve regarding contributing to the OH project.  I finally had time to look at some strange behavior, and I discovered that the linter installation did not work the way I thought it did.  I believe I now have things configured correctly, so you will see me going back through the entire OH1-addons fixing things again; hopefully for the last time.  The learning curve has been challenging, but I am getting there.  Slowly.

**Pull request merged**